<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RtmpLive</class>
 <widget class="QMainWindow" name="RtmpLive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RtmpLive</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>585</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>视频下载</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>450</y>
        <width>168</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="dstartButton1">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dstopButton1">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>501</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>信息</string>
      </property>
      <widget class="QTextBrowser" name="dinfoBrowser">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>461</width>
         <height>192</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(24, 58, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(80, 255, 250);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>477</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>地址：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="durlEdit">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dfileEdit">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>frame</zorder>
     <zorder>label</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget_2</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>视频推流</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(80, 255, 250);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>571</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>文件</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>400</y>
          <width>168</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="sfstartButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sfstopButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>501</width>
          <height>231</height>
         </rect>
        </property>
        <property name="title">
         <string>信息</string>
        </property>
        <widget class="QTextBrowser" name="sfinfoBrowser">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>461</width>
           <height>192</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(15, 39, 255);</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>465</width>
          <height>30</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sfileEdit">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>视频</string>
       </attribute>
       <widget class="QLabel" name="videolabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>480</width>
          <height>320</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_6">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>400</y>
          <width>168</width>
          <height>42</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="svstartButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="svstopButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QComboBox" name="videocomboBox">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>360</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>350</y>
          <width>271</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>信息</string>
        </property>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>54</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>时长：</string>
         </property>
        </widget>
        <widget class="QLabel" name="timerlabel">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>70</y>
           <width>171</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0);
font: 12pt &quot;Agency FB&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>54</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>警告：</string>
         </property>
        </widget>
       </widget>
       <widget class="QComboBox" name="dealcomboBox">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>360</y>
          <width>101</width>
          <height>30</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="livecheckBox">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>410</y>
          <width>51</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>远程</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>465</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>地址：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surlEdit">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>label_2</zorder>
     <zorder>tabWidget</zorder>
     <zorder>layoutWidget_3</zorder>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>190</width>
      <height>550</height>
     </rect>
    </property>
    <property name="title">
     <string>功能选择</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="downloadradioButton">
        <property name="text">
         <string>下载</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pushradioButton">
        <property name="text">
         <string>推流</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="playradioButton">
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
